test/builder_test.js

```diff
@@ -29,6 +29,9 @@ RSVP.on('error', error => {
   throw error;
 });
 
+function wait(time) {
```

Comment:
I like calling this `settled` because `await wait()` is weird 😝

Reply:
i think this is more `nextTick`